swiftnav/ambiguity_test_c.pxd

```diff
@@ -9,14 +9,27 @@
 
 
 from common cimport *
+cimport memory_pool_c
+cimport sats_management_c
 
 cdef extern from "libswiftnav/ambiguity_test.h":
+
+  ctypedef struct hypothesis_t:
+    s32 N[22]
```

Comment:
Yeah I should clean that up but not sure how. The cython definition is only for the cython -> c step and then the real header file is used for the actual build so it doesn't have to be correct. I guess I'll make a wrapper for constants.h